todo/repositories/user_repository.py

```
@@ -0,0 +1,55 @@
+    def _get_collection(cls):
+        return DatabaseManager().get_collection("users")
+
+    @classmethod
+    def get_by_id(cls, user_id: str) -> Optional[UserModel]:
+        try:
+            collection = cls._get_collection()
+            object_id = PyObjectId(user_id)
+            doc = collection.find_one({"_id": object_id})
+            return UserModel(**doc) if doc else None
+        except Exception:
+            raise GoogleUserNotFoundException()
```

Pylint (3.3.7)

[warning] Consider explicitly re-raising using 'except Exception as exc' and 'raise GoogleUserNotFoundException() from exc'

(W0707)